src/hzb_combinatorial_libraries/schema.py

```diff
@@ -752,4 +783,92 @@ def normalize(self, archive, logger: BoundLogger) -> None:
         super(UnoldThermalEvaporation, self).normalize(archive, logger)
 
 
+class PixelProperty(EntryData):
+    m_def = Section(
+        categories=[UnoldLabCategory],
+        label='UnoldPixelProperty'
+    )
+    conductivity = Quantity(
+        type=np.dtype(np.float64),
+        # unit='1/Ω',
+        shape=[],
+        description="""
+                The conductivity of the Pixel.
+                """,
+        a_eln=dict(
+            component='NumberEditQuantity')
+    )
+    bandgap = Quantity(
+        type=np.dtype(np.float64),
+        description='Band gap value in eV.',
+        a_eln=dict(component='NumberEditQuantity')
+    )
+
+    PLQY = Quantity(
+        type=np.dtype(np.float64),
+        # unit='%',
+        description='Energy integrated value of the PL spectrum.',
+        a_eln=dict(component='NumberEditQuantity')
+    )
+
+    implied_voc = Quantity(
+        type=np.dtype(np.float64),
+        # unit='V',
+        description='Estimated open circuit voltage based on PL measurements.',
+        a_eln=dict(component='NumberEditQuantity')
+    )
+
+    FWHM = Quantity(
+        type=np.dtype(np.float64),
+        # unit='eV', # todo check if this is correct
+        description='FWHM based on PL measurements.',
+        a_eln=dict(component='NumberEditQuantity')
+    )
+
+
+class Pixel(ContinuousCombiSample, EntryData, ArchiveSection):
+    m_def = Section(
+        categories=[UnoldLabCategory],
+        label='UnoldPixel'
+    )
+    thickness = Quantity(
+        type=np.dtype(np.float64),
+        unit='nm',
+        shape=[],
+        description="""
+            The thickness of the pixel.
+            """,
+        a_eln=dict(
+            component='NumberEditQuantity')
+    )
+    properties = SubSection(
+        section_def=PixelProperty,
+        description='''
+          The properties of the pixel.
+          ''',
+    )
+    samples = SubSection(
```

Comment:
why does this repeat? maybe add a label that it references the library.

Reply:
no need to repeat, other data types have samples also as a repeated section. `set_sample_references` in `baseclasses` also assume samples are a list.

Reply:
this is different form other data types, since those are activites, while here it is not. the name samples might also be confusing, since sample is a synonym for pixel in the continous library world as far as i know.

Reply:
i dont understand. so i change it to a name called lib_ref and not repeating ?